projects/plugins/crm/includes/ZeroBSCRM.MetaBoxes3.QuoteTemplates.php

```diff
@@ -155,8 +155,7 @@ public function html( $quoteTemplate, $metabox ) {
                         echo '<p style="text-align:right"><span class="ui basic blue label">'.esc_html__('Did you know: You can now use Quote Placeholders?','zero-bs-crm').' <a href="' . esc_url( $zbs->urls['kbquoteplaceholders'] ) . '" target="_blank">' . esc_html__('Read More','zero-bs-crm') . '</a></span></p>';
                         ##/WLREMOVE
 
-                        #http://stackoverflow.com/questions/3493313/how-to-add-wysiwyg-editor-in-wordpress-meta-box
-                        $content = zeroBSCRM_io_WPEditor_DBToWPEditor($quoteTemplateContent);
+						$content = $quoteTemplateContent; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
```

Comment:
This is a preexisting issue, but let's use `wp_kses()` to filter output from the database:

```suggestion
						$content = wp_kses( $quoteTemplateContent, $zbs->acceptable_html ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
```